com.unity.netcode.gameobjects/Runtime/Messaging/RpcParams.cs

```diff
@@ -2,7 +2,7 @@ namespace Unity.Netcode
 {
     public interface IHasUpdateStage
     {
-        public NetworkUpdateStage UpdateStage
+        NetworkUpdateStage UpdateStage
```

Comment:
Bad merge with develop?

Reply:
intentional (`public` shouldn't be there) but will move under a separate PR.